Formula/ace.rb

```diff
@@ -11,6 +14,9 @@ class Ace < Formula
     sha256 "72f21357797c350fe547a7978f402e564cf328c3662a0e321d1ec293f4899139" => :mavericks
   end
 
+  # this patch backports new files for macOS Sierra and possibility to compile with Xcode 8 on El Capitan
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file? Thanks!